src/coreclr/src/debug/runtimeinfo/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+set(RUNTIMEINFO_SOURCES
+    runtimeinfo.cpp
+)
+
+add_library_clr(runtimeinfo_lib STATIC ${RUNTIMEINFO_SOURCES})
+
+add_dependencies(runtimeinfo_lib runtime_module_index_header)
```

Comment:
I would prefer dropping the `_lib` suffix. We don't have any other library with such a naming convention.